devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -44,6 +45,9 @@ public class BuildMojo extends QuarkusBootstrapMojo {
     @Component
     MavenProjectHelper projectHelper;
 
+    @Component
+    BuildAnalyticsProvider analyticsProvider;
```

Comment:
The component is automatically closed? I'm surprised it is given it doesn't implement any marker interface.

Reply:
Closing is performed in the BootstrapSessionListener